components/movementsensor/gpsrtkpmtk/gpsrtkpmtk_test.go

```diff
@@ -199,14 +200,13 @@ func TestReconfigure(t *testing.T) {
 			NtripPass:            "somepass",
 			NtripUser:            "someuser",
 			NtripMountpoint:      "NYC",
-			Board:                testBoardName,
-			I2CBus:               testBusName,
-			I2CAddr:              testi2cAddr,
+			I2CBus:               testI2cBus,
+			I2CAddr:              testI2cAddr,
 			I2CBaudRate:          115200,
 		},
 	}
 	err := g.Reconfigure(context.Background(), nil, conf)
-	test.That(t, err, test.ShouldNotBeNil)
+	test.That(t, err, test.ShouldBeNil)
```

Comment:
FWIW, the error used to be that the board we depend on was not in the dependencies, because we were passing `nil` for the dependencies on line 208. I wonder if the test was supposed to do something else instead? I dunno. Reconfiguration no longer has errors.